charts/tyk-gateway/templates/hpa.yaml

```diff
@@ -0,0 +1,28 @@
+{{- if .Values.autoscaling.enabled }}
+apiVersion: autoscaling/v2beta1
```

Comment:
I think this api has reached v1 in v1.18 or so. So, don't need to use the beta version.